_pages/documentation/gem5-stdlib/8-extending-disk-images.md

```diff
@@ -0,0 +1,244 @@
+---
+
+layout: documentation  
+title: Extending Pre-Built Disk Images Using Packer  
+parent: gem5-standard-library  
```

Comment:
Why's this bundled in with the standard library? I don't think that's the right parent.